src/hooks/search/index.ts

```diff
@@ -47,67 +51,93 @@ const useSearch = (
 
     // TODO: Refactor to useSWRInfinite and implement client-side pagination.
     const { data, error, isValidating } = useSWR(key, fetcher, {
-        revalidateIfStale: false,
+        // revalidateIfStale: false,
```

Comment:
commenting, just in case this was an oversight

Reply:
so this was intentionally commented, but what's going on here is a bit fuzzy. my understanding is this:
- the `fallback` i.e. initial search data is only the first "page" of data and is baked into the build files by next
- when the page is loaded, `useSWR` initially returns the `fallback` data, but simultaneously runs the query to fetch the full up-to-date data
- `useSWR` switches the `fallback` data to the new up-to-date data once the query completes, which usually is very soon after the page loads

all of this is to say that if `revalidateIfStale` is false, it will never perform that initial search query and only the first page of data will be shown until the user interacts with one of the search components, effectively hiding the "Load more" button on first render. commenting this out doesn't seem to have any other side effects from what i can tell.